www-servers/pound/pound-3.0.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+MY_P=${P/p/P}
+DESCRIPTION="A http/https reverse-proxy and load-balancer"
+HOMEPAGE="https://www.apsis.ch/pound.html"
+SRC_URI="http://www.apsis.ch/pound/${MY_P}.tgz"
+
+LICENSE="GPL-3"
```

Comment:
Then their homepage says:

> the uthash library. A copy is included in the Pound distribution, but you may want to download the most up-to-date version. The copyright notice is here
> the hpack library. A copy is included in the Pound distribution, with adjustments for Linux compatability. The copyright is held by Reyk Floeter (BSD license).

So you'll have to include the licenses for these bundled libs too, **if** they're being used in here (please verify).

Reply:
Yes they are used